### PR TITLE
Allow editing of currently pending orgs - frontend

### LINKED ISSUE #2801: Frontend: Support editing orgs before id verification
### Description
Background in #2569. This is the frontend half of that ticket.

### Proposed solution 
We'll need to create a modal that supports editing and saving the new organization. The modal should pre-populate the existing org information and give Support the option to change each of the fields if necessary.

### Additional context
We don't have designs for this one, and should probably talk to Sharon at HHS Support about her team's workflow before changing this too drastically.
One frequently asked-for feature on this page is the ability to sort chronologically, and to have the organization admin's information present on the page.

### LINKED ISSUE #2863: Admin completing manual id verification from queued org should disable save button
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When completing manual id verification of an org in the new method (from a queued org), the "Save changes" button is not disabled and can be clicked multiple times.  Since the activation process will be much slower now, it is more important to block the button from being clicked.

https://user-images.githubusercontent.com/77013690/139541934-c6f501d4-d9bd-4eb5-96e7-94dbf4d3b440.mp4

### Steps To Reproduce

From the branch bmaden-usds/enable-org-queue (https://github.com/CDCgov/prime-simplereport/pull/2859)

1. Create a new queued org (no need to attempt id verification)
2. Go to the admin panel's "Organizations pending identify verification" page
3. Select the newly queued org
4. Click "Save changes" (more than once for fun)

### Expected Behavior

1. The admin should only be able to make the request once and then have to wait until the backend responds before clicking it again.
2. The button should be disabled again after successful verification (with no items selected after completion)

### Additional context

_No response_

### Does this issue have a priority label?

- [X] I have added a priority label

### Has issue been assigned to a team?

- [X] I have added a team label

### PR DESCRIPTION
**Pull request by nickclyde**
## Related Issue or Background Info

- Resolves #2801 
- Resolves #2863 
- Depends on #2800 

## Changes Proposed

- Add an edit button to each row in the ID verification table
- Clicking the button will open a modal with a form to edit the org's name and admin details
- Validate the form

## Additional Information

- decisions that were made
- discussion of tradeoffs / future work
- complaints about how bad the code is

## Screenshots / Demos

https://user-images.githubusercontent.com/9121162/140837536-a022c9a1-5c4e-4900-81d7-72eb6d31b067.mp4

### UI
- [ ] Any changes to the UI/UX are approved by design 
- [ ] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [ ] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [ ] Database changes are submitted as a separate PR
  - [ ] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [ ] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [ ] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [ ] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [ ] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [ ] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [ ] Any dependencies introduced have been vetted and discussed

## Cloud
- [ ] DevOps team has been notified if PR requires ops support
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Comment by nickclyde** [comment]
This now resolves #2863 as well

File: frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizationsContainer.test.tsx [hunk 1]
```diff
@@ -163,5 +213,102 @@ describe("PendingOrganizationsContainer", () => {
...
+            await userEvent.click(screen.getByText("Save"));
+          });
+          expect(
+            screen.getByText("A valid phone number is required", {
+              exact: false,
+            })
+          ).toBeInTheDocument();
+        });
+      });
+    });
```
Comment (bmaden-usds) [inline frontend/src/app/supportAdmin/PendingOrganizations/PendingOrganizationsContainer.test.tsx:312]: Thanks for the good test coverage!

**Review by bmaden-usds — APPROVED** [review]
LGTM!

Looking forward to seeing how this affects future support requests!

File: frontend/src/app/supportAdmin/PendingOrganizations/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  adminPhone: string | undefined;
+}
+
+const phoneNumberIsValidOrEmpty = (input: any) =>
+  input === "" || phoneNumberIsValid(input);
+
+export const pendingOrganizationSchema: yup.SchemaOf<PendingOrganizationFormValues> = yup
+  .object()
+  .shape({
+    name: yup.string().required("Organization name is required"),
```
Comment (emmastephenson) [inline frontend/src/app/supportAdmin/PendingOrganizations/utils.tsx:19]: org name isn't actually required for the mutation, so I don't think we need to validate on this one. They only have to enter the name if they want to change it

**Review by emmastephenson — APPROVED** [review]

**Pull request merged**